.github/workflows/build.yml

```diff
@@ -26,15 +36,13 @@ jobs:
           pip install pylint-django
           pip install pylint
           pip install djangorestframework
+          pip install coveralls python-dateutil
```

Comment:
Please move python-dateutil and djangorestframework to requirements.txt

Reply:
djangorestframework I moved in requirements.txt but python-dateutil has some problem if it move from requirements.txt so I didnt move it.